src/polymer/polymer-core-feature.ts

```diff
@@ -12,9 +12,21 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {ScannedPolymerElement} from '../polymer/polymer-element';
+import {Document, Feature, Resolvable, SourceRange, Warning} from '../model/model';
+import {ScannedPolymerProperty} from './polymer-element';
 
 /**
- * The metadata for a Polymer core feature.
+ * Scanned representation of a Polymer 1.x core "feature".
  */
-export interface ScannedPolymerCoreFeature extends ScannedPolymerElement {}
+export class ScannedPolymerCoreFeature extends Resolvable {
+  sourceRange: SourceRange|undefined;
+  warnings: Warning[] = [];
```

Comment:
I do need to re-declare them unless I change Resolved to be an abstract class instead of an interface. So this depends on the discussion above.